addons/addon-base-raas/packages/base-raas-services/lib/plugins/env-provisioning-plugin.js

```diff
@@ -220,6 +220,29 @@ async function updateEnvOnProvisioningSuccess({
       const environmentDnsService = await container.find('environmentDnsService');
       await environmentDnsService.createRecord('rstudio', envId, dnsName);
     }
+    else if (connectionTypeValue.toLowerCase() === 'rstudiov2') {
+      const albService = await container.find('albService');
+      const deploymentItem = await albService.getAlbDetails(requestContext, resolvedVars.projectId);
+      const albExists = await albService.checkAlbExists(requestContext, resolvedVars.projectId);
```

Comment:
Seems like as part of both the calls here we are calling `getAlbDetails`. Can we avoid that and do that just once.

Reply:
We will make this change